docs/porting/display.rst

```diff
@@ -232,6 +232,16 @@ If the performance monitor is enabled, the value of :c:macro:`LV_DEF_REFR_PERIOD
 consistent with the refresh period of the display to ensure that the statistical results are correct.
 
 
+Force refreshing
+----------------
+
+Normally the the invalidated areas (marked for redraw) are rendered in :cpp:func:`lv_timer_handler` in every
```

Comment:
Normally the the     ---->Normally the the 

It seems to have added an extra "the"

Reply:
Fixed, thank you